### PR TITLE
Documentation of variable length submessage limits [7413]

### PR DESCRIPTION
**Pull request by IkerLuengo**
Describes how to limit the size of variable length submessages: Partition, Userdata and Properties

DO NOT MERGE UNTIL THE CODE IS MERGED TOO

### CODE REVIEW

**Comment by richiware** [comment]
Build status: [Build Status](http://jenkins.eprosima.com:8080/job/FastRTPS%20Docs%20Manual/127/)

**Comment by richiware** [comment]
Build status: [Build Status](http://jenkins.eprosima.com:8080/job/FastRTPS%20Docs%20Manual/128/)

**Comment by richiware** [comment]
Build status: [Build Status](http://jenkins.eprosima.com:8080/job/FastRTPS%20Docs%20Manual/144/)

File: code/CodeTester.cpp [hunk 1]
```diff
@@ -165,6 +165,12 @@ participant_attr.rtps.allocation.participants = eprosima::fastrtps::ResourceLimi
 participant_attr.rtps.allocation.readers = eprosima::fastrtps::ResourceLimitedContainerConfig::fixed_size_configuration(2u);
 // We know we have at most 1 writer on each participant
 participant_attr.rtps.allocation.writers = eprosima::fastrtps::ResourceLimitedContainerConfig::fixed_size_configuration(1u);
+// We know the maximum size of partitiondata
+participant_attr.rtps.allocation.max_partitions = 256u;
+// We know the maximum size of user data
+participant_attr.rtps.allocation.max_user_data = 256u;
+// We know the maximum size of properties data
+participant_attr.rtps.allocation.max_properties = 512u;
```
Comment (MiguelCompany) [inline code/CodeTester.cpp]: All these parameters are inside `participant_attr.rtps.allocation.data_limits`

**Comment by richiware** [comment]
Build status: [Build Status](http://jenkins.eprosima.com:8080/job/FastRTPS%20Docs%20Manual/145/)

File: code/CodeTester.cpp [hunk 2]
```diff
@@ -165,6 +165,12 @@ participant_attr.rtps.allocation.participants = eprosima::fastrtps::ResourceLimi
 participant_attr.rtps.allocation.readers = eprosima::fastrtps::ResourceLimitedContainerConfig::fixed_size_configuration(2u);
 // We know we have at most 1 writer on each participant
 participant_attr.rtps.allocation.writers = eprosima::fastrtps::ResourceLimitedContainerConfig::fixed_size_configuration(1u);
+// We know the maximum size of partitiondata
```
Comment (MiguelCompany) [inline code/CodeTester.cpp]:
```suggestion
// We know the maximum size of partition data
```

**Comment by richiware** [comment]
Build status: [Build Status](http://jenkins.eprosima.com:8080/job/FastRTPS%20Docs%20Manual/146/)

**Pull request merged**